src/StoreKeeper/WooCommerce/B2C/Helpers/Seo/StorekeeperSeo.php

```diff
@@ -0,0 +1,268 @@
+            get_option(self::STOREKEEPER_SEO_OPTION_KEY)
+        );
+
+        $categories = &$termMeta['product_cat'];
+
+        if (isset($categories[$tag->term_id])) {
+            $category = $categories[$tag->term_id];
+        } else {
+            $category = [];
+        }
+
+        ?>
+        <tr class="form-field">
+            <th scope="row" valign="top">
+                <label for="cat_seo_title"><?php _e('SEO Title'); ?></label>
+            </th>
+            <td>
+                <input
+                        type="text" name="Cat_meta[seo_title]"
+                        id="Cat_meta[seo_title]" size="3" style="width:60%;"
+                        value="<?php echo esc_html($category['wpseo_title']) ?? ''; ?>"
+                        readonly
+                ><br />
+            </td>
+        </tr>
+        <tr class="form-field">
+            <th scope="row" valign="top">
+                <label for="cat_seo_description"><?php _e('SEO Description'); ?></label>
+            </th>
+            <td>
+                <input
+                        type="text" name="Cat_meta[seo_description]"
+                        id="Cat_meta[seo_description]" size="3" style="width:60%;"
+                        value="<?php echo esc_html($category['wpseo_desc']) ?? ''; ?>"
+                        readonly
+                ><br />
+            </td>
+        </tr>
+        <tr class="form-field">
+            <th scope="row" valign="top">
+                <label for="cat_seo_keywords"><?php _e('SEO Keywords'); ?></label>
+            </th>
+            <td>
+                <input
+                        type="text" name="Cat_meta[seo_keywords]"
+                        id="Cat_meta[seo_keywords]" size="3" style="width:60%;"
+                        value="<?php echo esc_html($category['wpseo_focuskw']) ?? ''; ?>"
+                        readonly
+                ><br />
+            </td>
+        </tr>
+        <?php
+    }
+
+    public function extraProductFields($post) {
+        $seoTitle = WordpressExceptionThrower::throwExceptionOnWpError(
+            get_post_meta($post->ID, 'wpseo_title', true)
+        );
+        $seoDescription = WordpressExceptionThrower::throwExceptionOnWpError(
+            get_post_meta($post->ID, 'wpseo_desc', true)
+        );
+        $seoKeywords = WordpressExceptionThrower::throwExceptionOnWpError(
+            get_post_meta($post->ID, 'wpseo_focuskw', true)
+        );
+
+        ?>
+        <table id="alg-product-input-fields-table" class="alg-product-input-fields-table">
+            <tr>
+                <th scope="row" vertical-align="top">
+                    <label for="post_seo_title"><?php _e('SEO Title'); ?></label>
+                </th>
+                <td>
+                    <input
+                            type="text" name="Post_meta[seo_title]"
+                            id="Post_meta[seo_title]" size="3" style="width:800%;"
```

Comment:
800%?

Reply:
it ends up looking like this at 800%
<img width="595" alt="Screenshot 2023-02-07 at 08 47 13" src="https://user-images.githubusercontent.com/121933575/217183385-c605fd6c-e8ea-498f-8575-30cafb370f35.png">